tests/lca/imagebasedupgrade/mgmt/upgrade/tests/e2e-upgrade-test.go

```diff
@@ -325,21 +334,55 @@ var _ = Describe(
 			}
 		})
 
-		It("upgrades the connected cluster", reportxml.ID("71362"), func() {
+		It("upgrades the connected cluster to a newer XY version", reportxml.ID("71362"), func() {
 			By("Check if the target cluster is connected")
 			connected, err := cluster.Connected(APIClient)
+			Expect(err).NotTo(HaveOccurred(), "error checking if the cluster is connected")
 
 			if !connected {
 				Skip("Target cluster is disconnected")
 			}
 
+			By("Check if seed and target have the same XY version")
+			upgradedClusterVersionXY, err := ClusterVersionXY(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion)
+			Expect(err).NotTo(HaveOccurred(), "error retrieving the XY portion of the seed cluster")
+
+			if upgradedClusterVersionXY == originalClusterVersionXY {
+				Skip("XY portion of the OCP version between seed and target clusters is identical")
+			}
+
 			if err != nil {
 				Skip(fmt.Sprintf("Encountered an error while getting cluster connection info: %s", err.Error()))
 			}
 
 			upgrade()
 		})
 
+		It("upgrades the connected cluster to a newer z-stream", reportxml.ID("79176"), func() {
+			By("Check if the target cluster is connected")
+			connected, err := cluster.Connected(APIClient)
+			Expect(err).NotTo(HaveOccurred(), "error checking if the cluster is connected")
+
+			if !connected {
+				Skip("Target cluster is disconnected")
+			}
+
+			By("Check if seed and target have the same XY version")
+			upgradedClusterVersionXY, err := ClusterVersionXY(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion)
+			Expect(err).NotTo(HaveOccurred(), "error retrieving the XY portion of the seed cluster")
+
+			if upgradedClusterVersionXY != originalClusterVersionXY {
+				Skip("XY portion of the OCP version between seed and target clusters is different")
+			}
+
+			upgrade()
+
+			ibu, err = lca.PullImageBasedUpgrade(APIClient)
+			Expect(err).NotTo(HaveOccurred(), "error pulling imagebasedupgrade resource")
+			Expect(ibu.Object.Spec.SeedImageRef.Version).To(Not(Equal(MGMTConfig.SeedClusterInfo.SeedClusterOCPVersion)),
+				"The target cluster has the same OCP version after IBU")
+		})
+
 		It("upgrades the disconnected cluster", reportxml.ID("71736"), func() {
```

Comment:
We need the same XY check that is on like 350 here

Reply:
The added zstream test test is connected.
We don't have one for disco. Still think we should add it?